x/docs/internal/types/codec.go

```diff
@@ -15,7 +15,7 @@ import (
 
 // RegisterCodec registers concrete types on wire codec
 func RegisterCodec(cdc *codec.Codec) {
-	cdc.RegisterConcrete(MsgShareDocument{}, fmt.Sprintf("%s/ShareDocument", ModuleName), nil)
+	cdc.RegisterConcrete(MsgShareDocument{}, fmt.Sprintf("%s/ShareDocument", "commercio"), nil)
```

Comment:
I think we can avoid using the `fmt.Sprintf` method at all, replacing it with a simple "commercio/ShareDocument` string